app/__init__.py

```diff
@@ -40,9 +41,23 @@
     print 'WARNING: The initialization vector has not been changed from the'
     print '         default. This is insecure, please generate a new one'
 
+assets = Environment(app)
+indexjs = Bundle('../js/index.js', filters='rjsmin', output='js/index.js')
```

Comment:
When this is run after a `docker-compose up`, the files are owned by root. It's not like we'll need to edit these files by hand, but they should be renamed to `*.min.js` and/or put into a separate directory.